lib/common/widgets/game_finish_dialog.dart

```diff
@@ -14,110 +14,114 @@ Future<void> showGameFinishDialog({
     barrierDismissible: false,
     builder: (context) => Dialog(
       backgroundColor: Colors.transparent,
-      child: Builder(builder: (context) {
-        final sortedItems = items.sortedByType();
+      child: Builder(
+        builder: (context) {
+          final sortedItems = items.sortedByType();
 
-        return ConstrainedBox(
-          constraints: const BoxConstraints(
-            maxWidth: 600,
-            maxHeight: 600,
-          ),
-          child: Container(
-            padding: const EdgeInsets.all(24),
-            decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(40),
-              border: Border.all(
-                width: 2,
-                color: FlutterGameChallengeColors.textStroke,
-              ),
-              color: FlutterGameChallengeColors.detailsBackground,
+          return ConstrainedBox(
+            constraints: const BoxConstraints(
+              maxWidth: 600,
```

Comment:
Should it have a min size?

Reply:
I would prefer to leave it like this, cause in this case it will be as large to fit the content